test/coverlet.core.tests/Coverage/InstrumenterHelper.cs

```diff
@@ -279,7 +301,7 @@ public static void GenerateHtmlReport(CoverageResult coverageResult, IReporter r
             reportFile = Path.Combine(dir.FullName, Path.ChangeExtension("report", reporter.Extension));
             File.WriteAllText(reportFile, reporter.Report(coverageResult));
             // i.e. reportgenerator -reports:"C:\git\coverlet\test\coverlet.core.tests\bin\Debug\netcoreapp2.0\Condition_If\report.cobertura.xml" -targetdir:"C:\git\coverlet\test\coverlet.core.tests\bin\Debug\netcoreapp2.0\Condition_If" -filefilters:+**\Samples\Instrumentation.cs
-            new Generator().GenerateReport(new ReportConfiguration(
+            Assert.True(new Generator().GenerateReport(new ReportConfiguration(
```

Comment:
@Danielpalme after upgrade to 3.1 started to fail to generate reports, is there a way to understand what's wrong?I get only bool result.

Reply:
Could you please try again with version 4.4.4 of `ReportGenerator.Core`.

Until 4.4.3 the structure was:
```
lib\netstandard2.0
lib\netcoreapp
```

With 4.4.4 I added:
```
lib\netcoreapp3.0
```

.NET Core 3.x seems to ignore DLLs in `lib\netcoreapp`.

Reply:
I'll try asap thanks

Reply:
Got same error but the dll seem ok(new one)
![image](https://user-images.githubusercontent.com/7894084/72686249-f0858400-3af2-11ea-994d-21f315f38de1.png)

Reply:
Will debug this myself tomorrow evening.

Reply:
Will debug this myself tomorrow evening.

Reply:
Will debug this myself tomorrow evening.

Reply:
No worries no rush thank you!

Reply:
Does the problem still exist? You resolved this issue and merged the change into master, that's why I'm asking.

Reply:
Yes still exists I merged because I use automatic generation of report only for debugging purpouse. So to move stuff forward I merged this fix.

Reply:
You can clone repo
```
dotnet pack
dotnet build
dotnet test
```
And you can try with this test https://github.com/tonerdo/coverlet/blob/master/test/coverlet.core.tests/Coverage/CoverageTests.cs#L114 enabling that row and it will generate report during test.

Reply:
Feel free to ping me if you're in throuble.

Reply:
I've sent a PR #696 which finally resolves the problem.
The Nuget behavior seems to have changed in .NET Core 3.x. Not all files in `lib\netcoreapp3.0` get copied during building the project.